pom.xml

```diff
@@ -28,16 +28,16 @@
         <!-- Remove when a new parent version with MTF is available -->
         <munit.input.directory>src/test/munit</munit.input.directory>
         <munit.output.directory>${basedir}/target/test-mule/munit</munit.output.directory>
-        <munit.extensions.maven.plugin.version>1.1.3</munit.extensions.maven.plugin.version>
-        <munit.version>2.3.15</munit.version>
+        <munit.extensions.maven.plugin.version>1.2.0-SNAPSHOT</munit.extensions.maven.plugin.version>
+        <munit.version>3.1.0-SNAPSHOT</munit.version>
+        <mtf.tools.version>1.2.0-SNAPSHOT</mtf.tools.version>
         <mavenResources.version>3.3.0</mavenResources.version>
         <docker.maven.plugin.version>0.41.0</docker.maven.plugin.version>
         <docker.skip>${skipTests}</docker.skip>
         <maven.helper.plugin.version>3.3.0</maven.helper.plugin.version>
-        <net.bytebuddy.version>1.14.5</net.bytebuddy.version>
 
         <ftp-connector.version>2.0.0-SNAPSHOT</ftp-connector.version>
-        <sftp-connector.version>1.6.3-SNAPSHOT</sftp-connector.version> <!-- SFTP v2 doesn't not use the file-commons project-->
+        <sftp-connector.version>2.1.0-SNAPSHOT</sftp-connector.version> <!-- SFTP v2 doesn't not use the file-commons project-->
```

Comment:
We should use the latest sftp version that uses the file commons

Reply:
Yes, we should use the support/1.6.x latest version

Reply:
If we use the 1.6.3 version the test is not working in the pipeline because I'm getting timeout error when test against Mule 4.6 and 4.7. That SFTP connector version is using a old file commons version 1.4.1 that uses cglib so It doesn't mater what version we use because neither of them are using this connector version. The 2XSFTP version is not using file common but uses the same solution that this file commons version, not ByteBuddy. My suggestion is release this connector and then include it in the otghet connectors like SFTP 1.6.x, FTP, FTPS, and FILE imn order to test all FILE FAMILY connectors, the you can replace the version of connectors that are using this connectors effectively

Reply:
Ok, but it sounds weird that we have a test for a version of SFTP that doesn't use file-commons. If we are going to do it just to unlock ourselves, please create a ticket to modify it in the future (it could be to remove the tests that use SFTP or to modify the 1.6.x support branch to include the cglib replacement, whether or not a release is made).